spark-statsd-reporter/pom.xml

```diff
@@ -27,6 +55,21 @@
       <version>${spark.version}</version>
       <scope>provided</scope>
```

Comment:
we also need to exclude `codahale` and `dropwizard` groups from Spark dependency to avoid clashing of transitive dependencies.

Reply:
Added the exclusions.